pkg/venafi/tpp/connector.go

```diff
@@ -493,7 +495,32 @@ func (c *Connector) RenewCertificate(renewReq *certificate.RenewalRequest) (requ
 	if renewReq.CertificateDN == "" {
 		return "", fmt.Errorf("failed to create renewal request: CertificateDN or Thumbprint required")
 	}
-
+	if renewReq.CertificateRequest != nil && renewReq.CertificateRequest.OmitSANs {
+		type field struct {
+			Name  string
+			Value *string
+		}
+		r := struct {
+			AttributeData []field
+		}{[]field{
+			{"X509 SubjectAltName DNS", nil},
+			{"X509 SubjectAltName IPAddress", nil},
+			{"X509 SubjectAltName RFC822", nil},
+			{"X509 SubjectAltName URI", nil},
+			{"X509 SubjectAltName OtherName UPN", nil},
+		}}
+		guid, err := c.configDNToGuid(renewReq.CertificateDN)
+		if err != nil {
+			return "", err
+		}
+		statusCode, _, _, err := c.request("PUT", urlResourceCertificate+urlResource(guid), r)
+		if err != nil {
+			return "", err
+		}
+		if statusCode != http.StatusOK {
+			return "", fmt.Errorf("can't clean SANs values for certificate on server side")
+		}
+	}
```

Comment:
Is this code covered by tests?

Reply:
Also it's not very clear for me what we're doing here. I think we need to add comment with short description.